plugins/woocommerce/i18n/states.php

```diff
@@ -1145,6 +1145,102 @@
 		'SN' => __( 'Sinoe', 'woocommerce' ),
 	),
 	'LU' => array(),
+	'MA' => array( // Moroccan regions.
+			'ma01'  => __( 'Tangier-Tétouan', 'woocommerce' ),
```

Comment:
It looks like all the lines in this array have [one too many tab indents](https://github.com/woocommerce/woocommerce/actions/runs/7026425717/job/19152598723?pr=41769)...